src/icon-manifest/supportedFolders.ts

```diff
@@ -1,33 +1,31 @@
 /* tslint:disable max-line-length */
-
 import { IExtensionCollection, IFolderExtension, FileFormat } from '../models/IExtension';
 
 export const extensions: IExtensionCollection<IFolderExtension> = {
   supported: [
-    { icon: 'aws', extensions: ['aws'], format: FileFormat.svg },
-    { icon: 'aws', extensions: ['aws'], dot: true, format: FileFormat.svg },
+    { icon: 'aws', extensions: ['aws', '.aws'], format: FileFormat.svg },
```

Comment:
I noticed that you are proposing the `format: "svg"` style for custom icons. Should this also be applied to the default extensions?